vmm/src/config.rs

```diff
@@ -717,7 +717,7 @@ impl RngConfig {
         // Split the parameters based on the comma delimiter
         let params_list: Vec<&str> = rng.split(',').collect();
 
-        let mut src_str: &str = "";
+        let mut src_str: &str = DEFAULT_RNG_SOURCE;
```

Comment:
Something's off in your commit message:
```
 vmm: config: Add RNG parsing unit test

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

 Please enter the commit message for your changes. Lines starting
```
There are 2 extra lines after the SoB.

Reply:
Thanks. Fixed. Probably related to to a squash!